### PR TITLE
Add Azure Database for PostgreSQL

### PR DESCRIPTION
**Pull request by inv-jishnu**
I have updated the `SetupDatabaseForAzure` document by adding Azure Database for PostgreSQL section to it.

### CODE REVIEW

**Review by scalar-boney — APPROVED** [review]
LGTM!

File: docs/SetupDatabaseForAzure.md [hunk 1]
```diff
@@ -48,3 +48,24 @@ You can choose Flexible or Single server deployment for Azure Database for MySQL
 Note:-
 
 * The [Storage auto-grow](https://docs.microsoft.com/en-gb/azure/mysql/concepts-pricing-tiers#storage-auto-grow) feature automatically increases storage without impacting the workload and is enabled by default in server configuration while creating an Azure Database for MySQL instance.
+
+
+## Azure Database for PostgreSQL
+
+In this section, you will create an Azure Database for PostgreSQL.
```
Comment (feeblefakie) [inline docs/SetupDatabaseForAzure.md:55]:
```suggestion
```
Just repeating the section title is meaningless.
If we should say something more, let's do it.

File: docs/SetupDatabaseForAzure.md [hunk 2]
```diff
@@ -48,3 +48,24 @@ You can choose Flexible or Single server deployment for Azure Database for MySQL
...
+
+### Steps
+
+You can choose Flexible or Single server deployment for Azure Database for PostgreSQL, follow the [Azure official guide](https://docs.microsoft.com/en-us/azure/postgresql/overview-postgres-choose-server-options) to learn more.
+* To create an Azure Database for PostgreSQL,
```
Comment (feeblefakie) [inline docs/SetupDatabaseForAzure.md:60]: It'a an action, right? Why starting with `To create`? Should it be `Create an ...`?

File: docs/SetupDatabaseForAzure.md [hunk 3]
```diff
@@ -48,3 +48,24 @@ You can choose Flexible or Single server deployment for Azure Database for MySQL
...
 * The [Storage auto-grow](https://docs.microsoft.com/en-gb/azure/mysql/concepts-pricing-tiers#storage-auto-grow) feature automatically increases storage without impacting the workload and is enabled by default in server configuration while creating an Azure Database for MySQL instance.
+
+
+## Azure Database for PostgreSQL
+
+In this section, you will create an Azure Database for PostgreSQL.
+
+### Steps
+
+You can choose Flexible or Single server deployment for Azure Database for PostgreSQL, follow the [Azure official guide](https://docs.microsoft.com/en-us/azure/postgresql/overview-postgres-choose-server-options) to learn more.
```
Comment (feeblefakie) [inline docs/SetupDatabaseForAzure.md:59]:
```suggestion
You need to choose Flexible or Single server deployment for Azure Database for PostgreSQL. Read the [Azure official guide](https://docs.microsoft.com/en-us/azure/postgresql/overview-postgres-choose-server-options) to learn more.
```
`You can choose` means, you can choose it or you don't neet choose it, so it's misleading.
If users need to choose one or the other, you should say, `you need to`.
I've said this several times.

File: docs/SetupDatabaseForAzure.md [hunk 4]
```diff
@@ -48,3 +48,24 @@ You can choose Flexible or Single server deployment for Azure Database for MySQL
...
+* To create an Azure Database for PostgreSQL, 
+  * For Single Server deployment, follow this [Azure official guide](https://docs.microsoft.com/en-us/azure/postgresql/quickstart-create-server-database-portal).
+  * For creating a flexible server deployment, follow this [Azure official guide](https://docs.microsoft.com/en-us/azure/postgresql/flexible-server/quickstart-create-server-portal).
```
Comment (feeblefakie) [inline docs/SetupDatabaseForAzure.md:62]: Inconsistent. `For Single Server deployment` and `For creating a flexible server deployment`.

File: docs/SetupDatabaseForAzure.md [hunk 5]
```diff
@@ -48,3 +48,24 @@ You can choose Flexible or Single server deployment for Azure Database for MySQL
...
+* (Optional) Configure advanced monitoring services with the [Azure official guide](https://docs.microsoft.com/en-us/azure/postgresql/concepts-monitoring), monitoring is enabled in Azure Database for PostgreSQL by default.
+* (Optional) To scale resources like vCores and storage after the deployment,
```
Comment (feeblefakie) [inline docs/SetupDatabaseForAzure.md:64]: Ditto.

File: docs/SetupDatabaseForAzure.md [hunk 6]
```diff
@@ -48,3 +48,24 @@ You can choose Flexible or Single server deployment for Azure Database for MySQL
...
+  * Follow the [Azure official guide](https://docs.microsoft.com/en-gb/azure/postgresql/concepts-pricing-tiers#scale-resources) for learning more about scaling single server resources.
```
Comment (feeblefakie) [inline docs/SetupDatabaseForAzure.md:65]: To learn, then it should `Read`. To do steps, it should `Follow`.  Which is correct?

**Review by feeblefakie — CHANGES_REQUESTED** [review]

File: docs/SetupDatabaseForAzure.md (hunk 1 above)
Comment (inv-jishnu) [reply docs/SetupDatabaseForAzure.md]: Thank you for the suggestion.
I have updated the section with additional information.

File: docs/SetupDatabaseForAzure.md (hunk 3 above)
Comment (inv-jishnu) [reply docs/SetupDatabaseForAzure.md]: Sorry for the mistake.
I have updated the sentence.

File: docs/SetupDatabaseForAzure.md (hunk 4 above)
Comment (inv-jishnu) [reply docs/SetupDatabaseForAzure.md]: Sorry for the mistake.
I have updated it.
Thank you.

File: docs/SetupDatabaseForAzure.md (hunk 6 above)
Comment (inv-jishnu) [reply docs/SetupDatabaseForAzure.md]: Sorry for the mistake
`Read` will be correct considering the document linked here.
I have updated the sentence.

**Review by feeblefakie — APPROVED** [review]

**Pull request merged**